database_schemas/cassandra/benchflow.cql

```diff
@@ -339,50 +386,52 @@ CREATE TABLE trial_process_duration (
 
 CREATE TABLE trial_cpu (
   cpu_num_data_points bigint,
-  cpu_avg float,
-  cpu_q1 float,
-  cpu_q2 float,
-  cpu_q3 float,
-  cpu_p95 float,
-  cpu_mean float,
-  cpu_ci095_min float,
-  cpu_ci095_max float,
-  cpu_me float,
-  cpu_median float,
-  cpu_mode list<float>,
-  cpu_min float,
-  cpu_max float,
-  cpu_sd float,
-  cpu_integral float,
+  cpu_avg double,
+  cpu_q1 double,
+  cpu_q2 double,
+  cpu_q3 double,
+  cpu_p95 double,
+  cpu_mean double,
+  cpu_ci095_min double,
+  cpu_ci095_max double,
+  cpu_me double,
+  cpu_median double,
+  cpu_mode list<double>,
+  cpu_min double,
+  cpu_max double,
+  cpu_sd double,
+  cpu_integral double,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY (experiment_id, trial_id)
+  container_id text,
+  PRIMARY KEY (experiment_id, container_id, trial_id)
```

Comment:
@VincenzoFerme When you perform a query with where clauses with Cassandra for some reason you can't specify a column value if you didn;t specify all previous ones as well in the primary key, meaning that if I had trial_id first I couldn't have a query with "where experiment =x and container_id=y", because it will give an error saying that I needed to specify trial_id as well if I wanted to specify container_id. It's part of how Cassandra does clustering.